crates/net/network/src/config.rs

```diff
@@ -169,6 +196,9 @@ impl NetworkConfigBuilder {
             executor: None,
             hello_message: None,
             head: None,
+            tx_gossip_disabled: false,
+            #[cfg(feature = "optimism")]
+            optimism_network_config: OptimismNetworkConfigBuilder { sequencer_endpoint: None },
```

Comment:
```suggestion
            optimism_network_config: OptimismNetworkConfigBuilder::default(),
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/149